pkg/testdata/models/apis/ec2/0000-00-00/generator.yaml

```diff
@@ -68,8 +68,11 @@ operations:
     output_wrapper_field_path: VpcEndpoint
 
 resources:
-  DHCPOptions:
-
+  DhcpOptions:
+    fields:
+      DHCPConfigurations.Values:
+        set:
+          - from: AttributeValue.Value
```

Comment:
You might consider adding:

```yaml
      method: CREATE
```

here in order to exercise the OpType passing down into the recursive SetResourceForXXX functions.